addons/mail/static/src/models/discuss/discuss.js

```diff
@@ -498,6 +498,7 @@ function factory(dependencies) {
         replyingToMessageOriginThreadComposer: one2one('mail.composer', {
             inverse: 'discussAsReplying',
             related: 'replyingToMessageOriginThread.composer',
+            readonly: true,
```

Comment:
remove "should be considered read-only." from the doc, since it is read-only now